### DISCUSSION TITLE
Love it!

### DISCUSSION THREAD

**Discussion by EDIflyer** [jordan-dalby/ByteStash#74] [General]
I came here after seeing your project featured on the  [DB Tech YouTube channel](https://www.youtube.com/watch?v=PdqibbRSbg4).  Thanks for making it so simple to setup, I've put it behind my existing Authelia/Nginx Proxy Manager setup and it works really well!

**Comment by jordan-dalby** [comment]
Hey, thanks for pointing that video out, I'll make sure to give a thanks to DB Tech. 
Enjoy!

**Comment by vlavrynovych** [comment]
I also tried this project after the same video. Going to keep it in my homelab for private use and for sharing.
I love the simplicity of UI and functionality.

@jordan-dalby , thank you for a great project.

**Comment by rob4226** [comment]
This is a really nice project! I really like the UI!  I stumbled across it from [https://selfh.st](https://selfh.st/apps/?tag=Code+Snippets) and set it up on my local lan for personal use and it works really well.  Nice job!
